test/project_types/script/script_project_test.rb

```diff
@@ -7,6 +7,29 @@ class ScriptProjectTest < MiniTest::Test
     def setup
       @context = TestHelpers::FakeContext.new
       @script_name = 'name'
+      @extension_point_type = 'ep_type'
+    end
+
+    def test_initialize
+      ScriptProject
+        .any_instance
+        .expects(:lookup_config)
+        .with('extension_point_type')
+        .returns(@extension_point_type)
+
+      ScriptProject
+        .any_instance
+        .expects(:lookup_config)
```

Comment:
It would be much better to not stub out a private method like this in this case.  It would be much better to set the values for the `ShopifyCli::Project's @config` value using https://apidock.com/ruby/Object/instance_variable_set.  Otherwise, what if there is a bug in `lookup_config`?